ext/sets_test.go

```diff
@@ -328,6 +330,119 @@ func TestSets(t *testing.T) {
 	}
 }
 
+func TestSetsMembershipRewriter(t *testing.T) {
```

Comment:
Up to you: maybe add some tests around macros.

Reply:
Done. Added for macros and enums.